packages/components/src/ui/color-picker/styles.ts

```diff
@@ -53,9 +84,24 @@ export const ColorfulWrapper = styled.div`
 	.react-colorful__pointer {
 		height: 16px;
 		width: 16px;
+		border: ${ CONFIG.borderWidthFocus } solid rgba( 255, 255, 255, 0 );
+		box-shadow: inset 0px 0px 0px ${ CONFIG.borderWidthFocus } #ffffff;
 	}
 
 	${ StyledField } {
 		margin-bottom: 0;
 	}
+
+	${ inputHeightStyle }
+`;
+
+export const DetailsControlButton = styled( Button )`
```

Comment:
It's a bit unfortunate that this is not a button that can be shown properly with just props. Why do we need to tweak instead of relying on composition of uncustomized Button (is the isSmall prop not enough?)

Reply:
The isSmall prop on buttons with just an icon creates a 36-pixel width per 24 pixels height. Here we want a 24x24 pixels button.

Reply:
> The isSmall prop on buttons with just an icon creates a 36-pixel width per 24 pixels height. Here we want a 24x24 pixels button.

Do you think we should fix it in the button's styles instead?

```
	&.is-small {
		height: 24px;
		line-height: 22px;
		padding: 0 8px;
		font-size: 11px;

		&.has-icon:not(.has-text) {
			padding: 0 8px;
			width: 24px;
		}
	}
```

Accoding to this style from the button's styles, it's meant to be `24px` maybe there's bug?

Reply:
Good point, I will try to understand if it is a bug or not that the button appears with 36px, but the padding and other styles suggest it is intentional.